### PR TITLE
[10.x.x] Misc. SDET backports

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

- Bped Remove extra utr args from standalone run tests #3541
- Bped [Yamato] Use revision variables in update jobs #3518 

---
### Testing status
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/10.x.x%252Fsdet%252Fmisc-backports/.yamato%252Fall-universal.yml%2523PR_Universal_2020.2/5588502/job

### CODE REVIEW

File: .yamato/config/universal_stereo.metafile [hunk 1]
```diff
@@ -8,14 +8,6 @@ test_platforms:
   - type: editmode
   - type: Standalone
     name: Standalone_cache
-    utr_flags:
-    - [all]: compilation-errors-as-warnings
```
Comment (kivistikliis) [inline .yamato/config/universal_stereo.metafile:12]: is removal of `compilation-errors-as-warnings` intentional?

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/054_Lighting_Attenuation/Lightmap-0_comp_light.exr.meta [hunk 2]
```diff
@@ -0,0 +1,96 @@
+fileFormatVersion: 2
```
Comment (kivistikliis) [inline TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/054_Lighting_Attenuation/Lightmap-0_comp_light.exr.meta:1]: Are the png, meta, and asset file changes coming with the backports?

File: .yamato/config/universal_stereo.metafile (hunk 1 above)
Comment (theo-pnv) [reply .yamato/config/universal_stereo.metafile:12]: Tbh I'm not sure about this one. I will add it back, thanks.

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/054_Lighting_Attenuation/Lightmap-0_comp_light.exr.meta (hunk 2 above)
Comment (theo-pnv) [reply TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/054_Lighting_Attenuation/Lightmap-0_comp_light.exr.meta:1]: Thanks for catching that!

**Review by sophiaaar — APPROVED** [review]

File: .yamato/config/universal_stereo.metafile (hunk 1 above)
Comment (theo-pnv) [reply .yamato/config/universal_stereo.metafile:12]: Thinking again, I'm really not sure if we need this argument at run step... As the name implies, it should be used during the build/compilation step/job.
I found this:
```
--compilation-errors-as-warnings
    Fail a test suite if script compilation errors occurs. Temporary, until random compilation errors during editor startup are fixed
```

**Pull request merged**